### ISSUE TITLE
Safari: buttons and links are missing blue focus outline

### ISSUE THREAD

**Issue by bennettcc** [NCIOCPL/ncids#617]
### Basic Information ###
- Issue: 
- click on language toggle on Safari - there is no focus outline 
-  search button next to a search bar is missing focus
- site alert, footer links are also missing focus outline
- page options, side menu, mega menu, back to top... same
- Desired Outcome: outline should be present

### Steps to Reproduce ###
1. Go to any page with translation on safari
2. Click toggle
3. No outline

### Screen Shots ###
_Add desired screen shots_

**Comment by sarinapadilla** [comment]
Closing this ticket, as Safari needs to have the tab highlight setting on.
